ActionSheet.ios.js

```diff
@@ -5,10 +5,20 @@ import { ActionSheetIOS, View } from 'react-native';
 
 export default class ActionSheet extends React.Component {
   render() {
-    return <View style={{ flex: 1 }}>{React.Children.only(this.props.children)}</View>;
+    return (
+      <View style={{ flex: 1 }}>
+        {React.Children.only(this.props.children)}
+        </View>
+    );
   }
 
   showActionSheetWithOptions(options, onSelect) {
-    ActionSheetIOS.showActionSheetWithOptions(options, onSelect);
+    const iosOptions = {
+      ...options,
+      title: options.title === null ? undefined : options.title,
```

Comment:
do we need an explicit null check here? What happens if you just set it to options.title? Does it fail?

Reply:
Yeah I had a native crash on iOS. I’ll see what happens again and post the crash.

Reply:
ok cool! In that case you can also do `options.title == null` which will cover both undefined / null cases. double equals is ok here and there 😉

Reply:
Ok I wasn’t sure about the == being ok. 

So in that case, are you good with 

`title: options.title || undefined,`

?

Reply:
It was strange. When title was undefined, it didn’t crash, but did crash only if title was explicitly set to null (which is why I coded it like I did so it was clear it was just when exactly null).